client/src/app/recipe/recipe.component.scss

```diff
@@ -1,3 +1,24 @@
-:host {
-  display: block;
+.recipe-details {
+  position: absolute;
+  top: 5%; // Display below the nav-menu
```

Comment:
Don't position it absolutely. If it's block then it should display within it's constraints under the nav bar.